packages/adminrouter/extra/src/test-harness/modules/generic_test_code/common.py

```diff
@@ -142,6 +150,23 @@ def generic_correct_upstream_request_test(
     assert req_data['request_version'] == http_ver
 
 
+def header_is_absent(headers, header_name):
```

Comment:
I already have a function parameter called `assert_headers_absent` and other functions do not follow the style you are suggesting (e.g. `generic_upstream_headers_verify_test`, generic_no_slash_redirect_test, etc...).

I now it may be suboptimal - patches welcome if you are up to the task :)